v2/dtos/common/metrics.go

```diff
@@ -21,7 +21,7 @@ type Metrics struct {
 // https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/MetricsResponse
 type MetricsResponse struct {
 	Versionable `json:",inline"`
-	Metrics     `json:",inline"`
+	Metrics     Metrics `json:"metrics"`
```

Comment:
Extra Metrics variable name in the struct? did not see it doing that way originally

Reply:
Originally? This is new V2 DTO.

See swagger here: 
https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/openapi/v2/app-functions-sdk.yaml#L64

Reply:
ok